registration/include/pcl/registration/ndt.h

```diff
@@ -315,6 +315,20 @@ namespace pcl
       computeHessian (Eigen::Matrix<double, 6, 6> &hessian,
                       PointCloudSource &trans_cloud);
 
+      /** \brief Compute hessian of probability function w.r.t. the transformation vector.
+        * \note Equation 6.13 [Magnusson 2009].
+        * \param[out] hessian the hessian matrix of the probability function w.r.t. the transformation vector
+        * \param[in] trans_cloud transformed point cloud
+        * \param[in] transform the current transform vector
+        */
+      void
+      computeHessian (Eigen::Matrix<double, 6, 6> &hessian,
+                      PointCloudSource &trans_cloud,
+                      Eigen::Matrix<double, 6, 1> &transform)
+      {
```

Comment:
```suggestion
      {
        pcl::utils::ignore(transform);
```

This also needs the header to be included